src/components/AddressBook/AddressBook.tsx

```diff
@@ -174,89 +145,95 @@ export const AddressBook: FunctionComponent<AddressBookProps> = ({
 
   return (
     <div {...props}>
-      <div className="mb-5">
-        <Dropdown
-          dropdownButtonText={isLocal ? "Local" : name}
-          className="border-gray-300 border rounded-none mb-2 p-3 bg-white"
-          classNameShared="w-full max-w-sm"
-        >
-          <DropdownItem
-            onClick={() => {
-              setIsLocal(true);
-              resetThirdPartyAPIEndpointResult();
-            }}
+      <div className="flex-1 flex-col">
+        <h3 className="font-ubuntu text-4xl text-cloud-900">Address Book</h3>
+        <p className="mb-5 text-cloud-900">Please select an address book to view. </p>
+      </div>
```

Comment:
omit this and leave it at application end (tt-web, creator)

Reply:
done.